docs/secure-connections/sauce-connect/proxy-tunnels.md

```diff
@@ -308,15 +308,12 @@ fe80::1%lo0 localhost
 Once the host file has been altered, start Sauce Connect Proxy with the added argument `--metrics-address tunnelmetrics.com:8080`. Then, on the machine hosting Sauce Connect Proxy, you will see the metrics served at `http://tunnelmetrics.com:8080/debug/vars`.
 
 ## Improving Performance
-During testing, your website or application may load resources (e.g., tracking services, images/videos, advertisements), which can impact page load times and even cause tests to fail. If these external assets are publicly available on the Internet, then they can be cached to speed up requests. If these are not needed at all for testing purposes, you can disable or redirect traffic to improve performance.
+During testing, your website or application may load resources (e.g., tracking services, images/videos, advertisements), which can impact page load times and even cause tests to fail. If these external assets are publicly available on the Internet, then they can be fetched directly without using a tunnel. If these are not needed at all for testing purposes, you can disable the traffic to improve performance.
 
-### Disable Traffic to External Resources
-You can improve your overall test performance by disabling these third-party resource calls. If you're using Sauce Connect Proxy, the additional network hops required to access external resources has the potential to slow test execution dramatically. To retrieve resources directly, you can use the `--tunnel-domains` and `--direct-domains` flags to control which domains Sauce Connect will access during the test. To blocklist traffic so it is immediately dropped, use the `--fast-fail-regexps` command.
+### Configuring Traffic to External Resources
+You can improve your overall test performance by disabling these third-party resource calls. If you're using Sauce Connect Proxy, the additional network hops required to access external resources have the potential to slow test execution dramatically. To retrieve resources directly, you can use the [`--direct-domains`](/dev/cli/sauce-connect-proxy/#--direct-domains) flag. To blocklist traffic so it is immediately dropped, use the [`--fast-fail-regexps`](/dev/cli/sauce-connect-proxy/#--fast-fail-regexps) command.
 
-See the following for more information:
-
-* [Sauce Connect Proxy Command Line Quick Reference Guide](/dev/cli/sauce-connect-proxy)
-* [How to Remove Third Party Resources](http://elementalselenium.com/tips/66-blacklist) by Dave Haeffner, Elemental Selenium
+See [How to Remove Third Party Resources](http://elementalselenium.com/tips/66-blacklist) for more information.
 
 ### Be Aware of How Sauce Connect Proxy Caches Traffic
```

Comment:
Hi @waggledans , you had requested I remove this section. Before I do, I wanted to double-check on whether this related material should be removed/changed as well: 

- https://docs.saucelabs.com/secure-connections/sauce-connect/security-authentication/#ssl-certificate-bumping
- https://docs.saucelabs.com/dev/cli/sauce-connect-proxy/#--no-ssl-bump-domains
- https://docs.saucelabs.com/secure-connections/sauce-connect/setup-configuration/specialized-environments/#ssl-bumping